src/content/fr/2019/seo.md

```diff
@@ -0,0 +1,421 @@
+
+Mettons en perspective l'importance du référencement naturel. En Avril 2019, l'industrie du référencement a regardé avec horreur et fascination [ASOS a signalé une baisse de revenues de 87%](https://www.bbc.co.uk/news/business-47877688) après une "année difficile". La marque a attribué ce résultat à une baisse de classement et de visibilité dans les moteurs de recherche survenue après le lancement de plus de 200 microsites et à des changements importants dans la navigation de leur site web, entre autres changements techniques. Ouch.
+
+L'objectif du chapitre SEO du web Almanac est d'analyser les éléments des sites web qui ont un impact sur l'exploration et l'indexation des contenus pour les moteurs de recherche et, par conséquence, sur leurs performances. Dans ce chapitre, nous allons voir dans quelle mesure les sites web les plus fréquentés sont prêts à offrir une excellente expérience aux utilisateurs et aux moteurs de recherche, et quels sont ceux qui ont encore du travail à faire.
+
+Notre étude se base sur des données de [Lighthouse](./methodology#lighthouse), de [Chrome UX Report](./methodology#chrome-ux-report), et l'analyse des balises HTML. Nous nous sommes concentrés sur les fondements du SEO comme la balise `<title>`, les différents types de liens HTML, le contenu et la vitesse de chargement, mais aussi d'autres aspects techniques du référencement à savoir l'indexation, les données structurées, l'internationalisation, et les pages accélérées pour mobile (AMP) à travers 5 millions de sites web.
+
+Nos métriques personnalisées fournissent des informations qui, jusqu'à présent, n'avaient pas été exposées auparavant. Nous sommes maintenant en mesure de faire des constats sur l'adoption et la mise en œuvre d'éléments tels que la balise link alternate hreflang, l'éligibilité des résultats enrichis, l'utilisation de la balise title et même la navigation basée sur les ancres pour les applications d'une seule page (SPA).
+
+<p class="note">Remarque: Nos données se limitent à l'analyse des pages d'accueil uniquement et n'ont pas été collectées à partir d'analyses de toutes les pages des sites web. Cela aura un impact sur de nombreuses mesures dont nous discuterons, nous avons donc ajouté toutes les limitations pertinentes à chaque fois que nous mentionnons une mesure personnalisée. En savoir plus sur ces limitations dans notre <a href="./methodology">méthodologie</a>.</p>
```

Comment:
Add `&nbsp;` before :
```suggestion
<p class="note">Remarque&nbsp;: Nos données se limitent à l'analyse des pages d'accueil uniquement et n'ont pas été collectées à partir d'analyses de toutes les pages des sites web. Cela aura un impact sur de nombreuses mesures dont nous discuterons, nous avons donc ajouté toutes les limitations pertinentes à chaque fois que nous mentionnons une mesure personnalisée. En savoir plus sur ces limitations dans notre <a href="./methodology">méthodologie</a>.</p>
```
Note lots of examples of this but only highlighting this first one.

Reply:
Remarque: Nos/Remarque: **n**os

Reply:
OK